### ISSUE TITLE
Missing BRAW file The Security-Scoped Bookmark could not be resolved.

### ISSUE THREAD

**Issue by tangierc** [latenitefilms/BRAWToolbox#99]
I just moved to a new (more capable) computer using the same drives for the project I am working on, but all of my BRAW clips are offline now. Could use an assist in figuring this out. I already tried the adjusting filmstrip mode per the troubleshooting tip. All of my usual drives are mounted. I used migration assistant to move my account. Before posting this I downloaded a fresh BRAW Toolbox (after uninstalling the migrated one) from my App Store account.

![Screenshot 2023-03-22 at 8 05 29 PM](https://user-images.githubusercontent.com/9577806/227091987-06b05460-428f-4be0-a1e9-2503c33a5ccb.png)

**Comment by tangierc** [comment]
I got part of the way there by granting access to the drives in global settings, but I don't have a thumbnail folder to delete in order to jumpstart the thumbnail redrawing. So the clips can be scrubbed and played, but they're showing up as missing in icon view initially.

**Comment by tangierc** [comment]
Spoke too soon. I closed the library last night. When I opened it this morning the clips were offline. All drives were connected . I tried to do the process of granting access to the drive again in the global settings, but couldn't get that far. Selecting a clip in the timeline yielded a BRAW effect in the inspector with no settings to choose from - just a showing that BRAW Toolbox effect was an applied effect; name of the effect only. No security-scoped warning on the clips in the browser. Just the missing effect icon .
![Screenshot 2023-03-23 at 6 41 51 AM](https://user-images.githubusercontent.com/9577806/227227682-800810b3-592b-40fc-809c-f331e0d10f83.png)
